src/workspacer.Bar/BarPluginConfig.cs

```diff
@@ -19,6 +20,11 @@ public class BarPluginConfig
 
         public Color DefaultWidgetForeground { get; set; } = Color.White;
         public Color DefaultWidgetBackground { get; set; } = Color.Black;
+        
+        public Color Transparant { get; set; } = Color.Lime;
```

Comment:
I don't think the name of this property is enough to convey it's meaning. Wouldn't using the same name as elsewhere (i.e. `TransparencyKey`) be more obvious?

Reply:
You're right, I updated it accordingly